### DISCUSSION TITLE
Bind actions to mouse wheel scroll

### DISCUSSION THREAD

**Discussion by SPYFF** [ghostty-org/ghostty#4169] [Q&A] [answered]
I'm coming from GnomeTerminal/Tilix where I often use the `Ctrl+WheelScrollUp` and `Ctrl+WheelScrollDown` for increase and decrease the text size. Its useful when you have multiple splits, you can make most of the available screen estate.

Is it possible to bind the `increase_font_size` and `decrease_font_size` to `Ctrl+`mouse wheels? So far I cannot find any way to do it, I would be happy with a workaround.

**Comment by rhodes-b** [comment, marked as answer]
I dont believe this is currently possible but sounds very closely related to #3848 

https://ghostty.org/docs/config/reference#keybind
gives a link to source of what is valid and I dont see any mouse buttons in that

**Reply by SPYFF** [reply]
Thanks for the pointers. Indedd, #3848 is what I looking for, upvoted for better visibility. Also closing this avoid dupes.
